src/errors/test-run/templates.js

```diff
@@ -207,5 +207,9 @@ export default {
         Action "${err.selectorName}" argument error:
 
         ${escapeHtml(err.errMsg)}
+    `),
+
+    [TYPE.requestedSizeExceedsMaximumSize]: err => markup(err, `
+        The requested size exceeds the available maximum size.
```

Comment:
```
        Unable to resize the window because the specified size exceeds the screen size. On macOS, a window cannot be larger than the screen.
```